src/style.css

```diff
@@ -0,0 +1,101 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Avenir", Arial, Helvetica, sans-serif;
+  font-weight: 300;
```

Comment:
always set font styles to the `html` tag and inherit where needed

Reply:
ok, thanks